docs/giftcards.md

```diff
@@ -0,0 +1,32 @@
+---
+id: giftcards
+title: Accepting Gift Cards
+---
+Accept gift cards using the Square Payment Form.
+
+https://developer.squareup.com/docs/payment-form/gift-cards/part-1
+
+> **Important**: If you are adding Gift Cards to a payment page that already uses an instance of the Square Payment Form for credit cards, you **must** initialize a second Square Payment Form for the gift cards. How to [Render Multiple Payment Forms](customization.md#render-multiple-payment-forms)
+
+Gift cards are *not* supported in Sandbox at this time. See the docs for more information.
```

Comment:
Square Gift Card testing value was adding relatively recently https://developer.squareup.com/docs/testing/test-values#card-present-success-state-values

Reply:
Oh awesome! We need to update this doc then if it's not already in progress: https://developer.squareup.com/docs/payment-form/gift-cards/part-1#before-you-start